### ISSUE TITLE
[Bug] New version of node doesn't start up vpn client container

### ISSUE THREAD

**Issue by dsmits** [vantage6/vantage6#409]
**Describe the bug**
Today I noticed that the vpn client container is not running when a node has started up. I'm thinking this is something that might have been introduced in version `3.5.0` since the node image for that has been release today

**To Reproduce**
1. Make sure you have vantage6 version `3.5.0`
2. Run `vnode start etc.....`
3. Do `docker ps`

Log:
```bash
dsmits@mininode1:~$ sudo vnode start --attach -c /app/dummy.yaml 
[info]  - Starting node...
[info]  - Finding Docker deamon
[info]  - Checking that data and log dirs exist
[info]  - Pulling latest node image 'harbor2.vantage6.ai/infrastructure/node:petronas'
[debug] - Local image is up-to-date: harbor2.vantage6.ai/infrastructure/node:petronas
[info]  -  ... success!
[info]  - Creating Docker data volume
[info]  - Creating file & folder mounts
[info]  - Setting up databases
[info]  -   Processing database 'default:/app/data.csv'
[debug] -   - file-based database added
[info]  -   Processing database 'parquet:/app/data.parquet'
[debug] -   - file-based database added
[info]  - Running Docker container
[info]  - Success! container id = <Container: a0b8dc0853>
2022-11-30 10:38:53 - context        - INFO     - ---------------------------------------------
2022-11-30 10:38:53 - context        - INFO     -  Welcome to
2022-11-30 10:38:53 - context        - INFO     -                   _                     __  
2022-11-30 10:38:53 - context        - INFO     -                  | |                   / /  
2022-11-30 10:38:53 - context        - INFO     - __   ____ _ _ __ | |_ __ _  __ _  ___ / /_  
2022-11-30 10:38:53 - context        - INFO     - \ \ / / _` | '_ \| __/ _` |/ _` |/ _ \ '_ \ 
2022-11-30 10:38:53 - context        - INFO     -  \ V / (_| | | | | || (_| | (_| |  __/ (_) |
2022-11-30 10:38:53 - context        - INFO     -   \_/ \__,_|_| |_|\__\__,_|\__, |\___|\___/ 
2022-11-30 10:38:53 - context        - INFO     -                             __/ |           
2022-11-30 10:38:53 - context        - INFO     -                            |___/            
2022-11-30 10:38:53 - context        - INFO     - 
2022-11-30 10:38:53 - context        - INFO     -  --> Join us on Discord! https://discord.gg/rwRvwyK
2022-11-30 10:38:53 - context        - INFO     -  --> Docs: https://docs.vantage6.ai
2022-11-30 10:38:53 - context        - INFO     -  --> Blog: https://vantage6.ai
2022-11-30 10:38:53 - context        - INFO     - ------------------------------------------------------------
2022-11-30 10:38:53 - context        - INFO     - Cite us!
2022-11-30 10:38:53 - context        - INFO     - If you publish your findings obtained using vantage6, 
2022-11-30 10:38:53 - context        - INFO     - please cite the proper sources as mentioned in:
2022-11-30 10:38:53 - context        - INFO     - https://vantage6.ai/vantage6/references
2022-11-30 10:38:53 - context        - INFO     - ------------------------------------------------------------
2022-11-30 10:38:53 - context        - INFO     - Started application vantage6 with environment application
2022-11-30 10:38:53 - context        - INFO     - Current working directory is '/'
2022-11-30 10:38:53 - context        - INFO     - Successfully loaded configuration from '/mnt/config/dummy.yaml'
2022-11-30 10:38:53 - context        - INFO     - Logging to '/mnt/log/dummy.log'
2022-11-30 10:38:53 - context        - INFO     - Common package version '3.5.0'
2022-11-30 10:38:53 - context        - INFO     - vantage6 version '3.5.0'
2022-11-30 10:38:53 - context        - INFO     - Node package version '3.5.0'
2022-11-30 10:38:53 - node           - INFO     - Connecting server: https://v6server.carrier-mu.src.surf-hosted.nl:443/api
2022-11-30 10:38:53 - node           - DEBUG    - Authenticating
2022-11-30 10:38:53 - client         - DEBUG    - Authenticating node...
2022-11-30 10:38:53 - client         - INFO     - Successfully authenticated
2022-11-30 10:38:53 - client         - DEBUG    - Making request: GET | https://v6server.carrier-mu.src.surf-hosted.nl:443/api/node/2 | None
2022-11-30 10:38:53 - client         - DEBUG    - Making request: GET | https://v6server.carrier-mu.src.surf-hosted.nl:443/api/organization/2 | None
2022-11-30 10:38:54 - node           - INFO     - Node name: node_commodity - collab Node
2022-11-30 10:38:54 - client         - DEBUG    - Making request: GET | https://v6server.carrier-mu.src.surf-hosted.nl:443/api/collaboration/1 | None
2022-11-30 10:38:54 - node           - WARNING  - Enabling encryption!
2022-11-30 10:38:54 - encryption     - DEBUG    - Loading private key
2022-11-30 10:38:54 - client         - DEBUG    - Making request: GET | https://v6server.carrier-mu.src.surf-hosted.nl:443/api/organization/2 | None
2022-11-30 10:38:54 - client         - INFO     - Public key matches the server key! Good to go!
2022-11-30 10:38:54 - node           - INFO     - Setting up proxy server
2022-11-30 10:38:54 - node           - INFO     - Starting proxyserver at 'proxyserver:80'
2022-11-30 10:38:54 - node           - DEBUG    - Creating websocket connection with the server
2022-11-30 10:38:54 - engineio.client - WARNING  - WebSocket upgrade failed: connection error
2022-11-30 10:38:54 - node           - INFO     - Connected to host=https://v6server.carrier-mu.src.surf-hosted.nl on port=443
2022-11-30 10:38:54 - docker.utils.config - DEBUG    - Trying paths: ['/root/.docker/config.json', '/root/.dockercfg']
2022-11-30 10:38:54 - docker.utils.config - DEBUG    - No config file found
2022-11-30 10:38:54 - docker.utils.config - DEBUG    - Trying paths: ['/root/.docker/config.json', '/root/.dockercfg']
2022-11-30 10:38:54 - docker.utils.config - DEBUG    - No config file found
2022-11-30 10:38:54 - network_man..  - INFO     - Network vantage6-dummy-user-net already exists! Using existing network
2022-11-30 10:38:54 - node           - INFO     - Setting up VPN client container
2022-11-30 10:38:54 - docker.utils.config - DEBUG    - Trying paths: ['/root/.docker/config.json', '/root/.dockercfg']
2022-11-30 10:38:54 - docker.utils.config - DEBUG    - No config file found
2022-11-30 10:38:54 - docker.utils.config - DEBUG    - Trying paths: ['/root/.docker/config.json', '/root/.dockercfg']
2022-11-30 10:38:54 - docker.utils.config - DEBUG    - No config file found
2022-11-30 10:38:54 - vpn_manager    - DEBUG    - Used VPN images:
2022-11-30 10:38:54 - vpn_manager    - DEBUG    -   Alpine: harbor2.vantage6.ai/infrastructure/alpine
2022-11-30 10:38:54 - vpn_manager    - DEBUG    -   Client: harbor2.vantage6.ai/infrastructure/vpn-client
2022-11-30 10:38:54 - vpn_manager    - DEBUG    -   Config: harbor2.vantage6.ai/infrastructure/vpn-configurator
2022-11-30 10:38:54 - node           - DEBUG    - Using existing config file to connect to VPN
2022-11-30 10:38:54 - vpn_manager    - DEBUG    - Mounting VPN configuration file
2022-11-30 10:38:54 - addons         - WARNING  - Removing container that was already running: vantage6-dummy-vpn-client
2022-11-30 10:38:54 - node           - INFO     - Node <node_commodity - collab Node> joined room <all_connections>
2022-11-30 10:38:54 - node           - INFO     - Node <node_commodity - collab Node> joined room <all_nodes>
2022-11-30 10:38:54 - node           - INFO     - Node <node_commodity - collab Node> joined room <collaboration_1>
2022-11-30 10:38:54 - node           - INFO     - Node <node_commodity - collab Node> joined room <node_2>
2022-11-30 10:38:54 - node           - INFO     - (Re)Connected to the /tasks namespace
2022-11-30 10:38:54 - client         - DEBUG    - Making request: GET | https://v6server.carrier-mu.src.surf-hosted.nl:443/api/result | {'state': 'open', 'include': 'task', 'node_id': 2}
2022-11-30 10:38:54 - vpn_manager    - DEBUG    - Starting VPN client container
2022-11-30 10:38:54 - client         - INFO     - Decrypting input
2022-11-30 10:38:54 - encryption     - INFO     - Decrypted shared key: b',\xd7\xc5:yu\x89\x03G$\xed/\xdb\x8d\xd9D\xdb\xc7\xe0-\xa6"\xc0R\x84\xb2\xa8r\x18\xd2\x08\xf2'
2022-11-30 10:38:54 - node           - DEBUG    - [{'result': None, 'id': 27, 'ports': [], 'finished_at': None, 'task': {'name': 'test_echo', 'database': 'default', 'complete': False, 'init_user': 5, 'id': 18, 'image': 'harbor2.vantage6.ai/algorithms/n2n-diagnostics', 'run_id': 9, 'description': 'test', 'parent': None, 'children': [{'id': 19, 'link': '/api/task/19', 'methods': ['GET', 'DELETE']}], 'init_org': 5, 'collaboration': {'id': 1, 'link': '/api/collaboration/1', 'methods': ['DELETE', 'GET', 'PATCH']}}, 'assigned_at': '2022-11-30T10:35:11.945579+00:00', 'organization': {'id': 2, 'link': '/api/organization/2', 'methods': ['GET', 'PATCH']}, 'log': None, 'node': {'name': 'node_commodity - collab Node', 'ip': '10.155.40.207', 'status': 'online', 'id': 2}, 'input': b'\x80\x04\x95A\x00\x00\x00\x00\x00\x00\x00}\x94(\x8c\x06method\x94\x8c\x04echo\x94\x8c\x06master\x94\x88\x8c\x06kwargs\x94}\x94\x8c\x0bother_nodes\x94]\x94(K\x03K\x04esu.', 'started_at': '2022-11-30T10:35:12.011373+00:00'}]
2022-11-30 10:38:54 - node           - INFO     - received 1 tasks
2022-11-30 10:38:54 - node           - DEBUG    - Tasks synced again with the server...
2022-11-30 10:38:54 - node           - DEBUG    - Could not connect to VPN.
2022-11-30 10:38:54 - node           - DEBUG    - Exception: 400 Client Error: Bad Request ("failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: "/app/entry.sh": permission denied: unknown")
2022-11-30 10:38:54 - node           - DEBUG    - Refreshing VPN keypair...
2022-11-30 10:38:54 - client         - DEBUG    - Making request: POST | https://v6server.carrier-mu.src.surf-hosted.nl:443/api/vpn/update | None
2022-11-30 10:38:55 - vpn_manager    - DEBUG    - Mounting VPN configuration file
2022-11-30 10:38:55 - addons         - WARNING  - Removing container that was already running: vantage6-dummy-vpn-client
2022-11-30 10:38:55 - vpn_manager    - DEBUG    - Starting VPN client container
2022-11-30 10:38:55 - node           - DEBUG    - Could not connect to VPN.
2022-11-30 10:38:55 - node           - DEBUG    - Exception: 400 Client Error: Bad Request ("failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: "/app/entry.sh": permission denied: unknown")
2022-11-30 10:38:55 - node           - DEBUG    - Requesting new VPN configuration file...
2022-11-30 10:38:55 - client         - DEBUG    - Making request: GET | https://v6server.carrier-mu.src.surf-hosted.nl:443/api/vpn | None
2022-11-30 10:38:56 - vpn_manager    - DEBUG    - Mounting VPN configuration file
2022-11-30 10:38:56 - addons         - WARNING  - Removing container that was already running: vantage6-dummy-vpn-client
2022-11-30 10:38:56 - vpn_manager    - DEBUG    - Starting VPN client container
2022-11-30 10:38:57 - node           - DEBUG    - Could not connect to VPN.
2022-11-30 10:38:57 - node           - DEBUG    - Exception: 400 Client Error: Bad Request ("failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: "/app/entry.sh": permission denied: unknown")
2022-11-30 10:38:57 - node           - DEBUG    - Setting up the docker manager
2022-11-30 10:38:57 - docker_manager - DEBUG    - Initializing DockerManager
2022-11-30 10:38:57 - docker.utils.config - DEBUG    - Trying paths: ['/root/.docker/config.json', '/root/.dockercfg']
2022-11-30 10:38:57 - docker.utils.config - DEBUG    - No config file found
2022-11-30 10:38:57 - docker.utils.config - DEBUG    - Trying paths: ['/root/.docker/config.json', '/root/.dockercfg']
2022-11-30 10:38:57 - docker.utils.config - DEBUG    - No config file found
2022-11-30 10:38:57 - docker_manager - INFO     - Copying /mnt/default.csv to /mnt/data
2022-11-30 10:38:57 - docker_manager - INFO     - Copying /mnt/parquet.csv to /mnt/data
ta.parquet'
[debug] -   - file-based database added
[info]  - Running Docker container
[info]  - Success! container id = <Container: 56f17521e2>
dsmits@vnode3:~$ sudo docker ps
CONTAINER ID   IMAGE                                              COMMAND                  CREATED         STATUS         PORTS              NAMES
56f17521e26b   harbor2.vantage6.ai/infrastructure/node:petronas   "vnode-local start -…"   5 seconds ago   Up 4 seconds   80/tcp, 2222/tcp   vantage6-dummy-user
```

**Comment by dsmits** [comment]
Hm I have the same problem now after reverting to `3.4.3` so maybe it's not about the specific version

**Comment by bartvanb** [comment]
This has been solved by #413
